test/getDeployedEntitiesStreamFromSnapshot.spec.ts

```diff
@@ -189,39 +171,28 @@ test('when successfully process a snapshot file', ({ components, stubComponents
     const { storage } = stubComponents
     storage.storeStream.callThrough()
     storage.retrieve.callThrough()
-    const processedSnapshots = createProcessedSnapshotsComponent(components)
-    const endStreamOfSpy = jest.spyOn(processedSnapshots, 'endStreamOf')
     const servers = [await components.getBaseUrl()]
     const r = []
     const stream = getDeployedEntitiesStreamFromSnapshot(
       {
         metrics: components.metrics,
-        fetcher: components.fetcher,
-        downloadQueue: components.downloadQueue,
         logs: components.logs,
-        storage: components.storage,
-        processedSnapshotStorage: components.processedSnapshotStorage,
-        snapshotStorage: components.snapshotStorage,
-        processedSnapshots
+        storage: components.storage
       },
       {
         requestRetryWaitTime: 0,
         requestMaxRetries: 10,
         tmpDownloadFolder: contentFolder
       },
-      {
-        snapshotHash: downloadedSnapshotFile,
-        greatestEndTimestamp: 9,
-        replacedSnapshotHashes: [],
-        servers: new Set(servers)
-      }
+      downloadedSnapshotFile,
+      new Set(servers)
     )
 
     for await (const deployment of stream) {
       r.push(deployment)
     }
-    expect(endStreamOfSpy).toBeCalledTimes(1)
-    expect(endStreamOfSpy).toBeCalledWith(downloadedSnapshotFile, 9)
+    // expect(endStreamOfSpy).toBeCalledTimes(1)
+    // expect(endStreamOfSpy).toBeCalledWith(downloadedSnapshotFile, 9)
```

Comment:
Is this test covered by the new ones added? should we remove it or is it tech debt?

Reply:
test removed